src/Converters.ts

```diff
@@ -18,9 +18,41 @@ export function fromRpcHttp(rpcHttp: rpc.IRpcHttp): RequestProperties {
     rawBody: fromTypedData(<rpc.ITypedData>rpcHttp.rawBody, false),
   };
 
+  if (rpcHttp.identities) {
+    httpContext.user = rpcHttp.identities.map(fromRpcClaimsIdentity);
+  }
+
   return httpContext;
 }
 
+export function fromRpcClaimsIdentity(rpcIdentity: IRpcClaimsIdentity): ClaimsIdentity {
+  const identity: ClaimsIdentity = {
+    authenticationType: fromNullableString(rpcIdentity.authenticationType),
+    nameClaimType: fromNullableString(rpcIdentity.nameClaimType),
+    roleClaimType: fromNullableString(rpcIdentity.roleClaimType),
+    claims: rpcIdentity.claims ? rpcIdentity.claims.map(fromRpcClaim) : [],
+  };
+
+  return identity;
+}
+
+export function fromRpcClaim(rpcClaim: IRpcClaim): Claim {
+  const claim: Claim = {
+    value: rpcClaim.value || undefined,
+    type: rpcClaim.type || undefined,
+  };
+
+  return claim;
+}
+
+export function fromNullableString(nullableString: INullableString | null | undefined): string | undefined {
+  if (typeof(nullableString) === 'undefined' || nullableString === null) {
+    return undefined;
+  } else {
+    return nullableString.value || undefined;
```

Comment:
Sort of. `undefined`, `null`, `""` and other "falsy" values evaluate to false in a conditional expression, and referencing a property that doesn't exist returns `undefined`. `obj.prop || <default>` is shorthand for "use the object property unless it's a falsy value or doesn't exist, in which case use <default>".

I was wrestling with getting TypeScript to behave at the time; after a little time to think about it this is messier and less precise than we probably want. Will revise.

Reply:
@mhoeger, @pragnagopa Per our conversation about Interfaces.ts, `fromNullableString`'s behavior is an internal implementation detail. In your opinion, should its return types be `string | null | undefined` or just `string | undefined`?

Reply:
@kashimiz - i think it should be `string | undefined`. Did a little digging, and I think that the generated types should really just be `string | undefined` instead of `string | null | undefined` (see [comment](https://github.com/dcodeIO/protobuf.js/issues/826#issuecomment-307754875)).

In Interfaces.ts, this will also let us just define claims identity properties as: `roleClaimType?: string`, which is implicitly `string | undefined`

Reply:
I don't know that we have control over the generated types (if you're referring to those built with `pbjs` from Azure/azure-functions-language-worker-protobuf),. Every field is `<something>|null`. (I would love to be wrong about this.) But I can change the converter methods.

Reply:
Sorry I didn't explain that well! I was wondering if `string | null | undefined` would be better practice because that's how every interface field is defined in the pbts output (`<something> | null | undefined`), but it looks like a potential bug in pbts that has resulted in more confusion. Long story short, I think `string | undefined` is best and changing the converter method and Interface.ts sound great!